tests/positive/command.cpp

```diff
@@ -133,6 +133,52 @@ TEST_F(PositiveCommand, ClearAttachmentsCalledInSecondaryCB) {
     m_commandBuffer->end();
 }
 
+TEST_F(PositiveCommand, ClearAttachmentsCalledWithoutFbInSecondaryCB) {
+    TEST_DESCRIPTION(
+        "Verify calling vkCmdClearAttachments inside secondary commandbuffer without linking framebuffer pointer to"
+        "inheritance struct");
+
+    ASSERT_NO_FATAL_FAILURE(Init());
+
```

Comment:
Testing is passing on all Android devices except the ShieldTV.

```suggestion
    if (IsPlatform(kShieldTV) || IsPlatform(kShieldTVb)) {
        GTEST_SKIP() << "This test should not run on Shield TV";
    }
```